data/blog/eleventh-wave-of-nostr-grants.mdx

```diff
@@ -0,0 +1,173 @@
+---
+title: "Eleventh Wave of Nostr Grants"
+date: '2025-04-16'
+tags: ['OpenSats', 'grants', 'nostr', 'wave']
+authors: ['default', 'arvin', 'julian']
+images: ['/static/images/blog/76-eleventh-wave-of-nostr-grants.jpg']
+draft: false
+summary: "OpenSats is funding five more open-source projects advancing the nostr ecosystem."
+---
+
+We're pleased to announce our eleventh wave of nostr grants. This round includes
+support for projects focused on decentralized live streaming, off-grid
+connectivity, web-of-trust infrastructure, private messaging, and open tools for
+game development. These efforts represent a growing ecosystem of builders
+working to expand nostr's capabilities while strengthening censorship resistance
+and permissionless access.
+
+The five projects receiving support in this wave are:
+
+- [Swae](#swae)
+- [HAMSTR](#hamstr)
+- [Vertex](#vertex)
+- [Secure Direct Messages](#secure-direct-messages)
```

Comment:
It would probably less confusing to call this one "Nostr Double Ratchet"

https://github.com/mmalmi/nostr-double-ratchet